Backend/app/common/PropertiesManager.py

```diff
@@ -81,8 +81,6 @@ def _set_attributes(self, config_section: str):
         for key, value in self.config.items(config_section):
             if value == "":
                 value = None
-            if value is None:
-                properties_manager_logger.warning(f"Using None for {key} in {config_section}")
```

Comment:
Unify this statement with the one above. Something like if value=="" or None. Then inside join both value=None and the new log message